packages/hooks/use-global-config/index.ts

```diff
@@ -10,7 +11,9 @@ export function useGlobalConfig(): ConfigProviderContext
 export function useGlobalConfig(key?: keyof ConfigProviderContext) {
   const config = inject(configProviderContextKey, {})
   if (key) {
-    return toRef(config, key)
+    return isObject(config) && hasOwn(config, key)
+      ? toRef(config, key)
+      : ref(undefined)
```

Comment:
I think `ref()` by default returns `Ref<undefined>`

Reply:
ref() returns `Ref<any>`, so we should pass `undefined` to ref()

Reply:
Fair enough, since `ref(undefined)` or `ref<undefined>()` are both workable way, we can accept any of them.